addons/website/static/src/scss/primary_variables.scss

```
@@ -1937,6 +2063,24 @@ $o-theme-color-palettes-compatibility-indexes: null; // Default to the color one
 // Create the list of palettes proposed by the UI
 $o-selected-color-palettes-names: (
     // Proposed by the configurator
+    'light-1',
```

I'd rename the new ones you're in introducing using the current default platette nomenclature, so:

```suggestion
    'default-light-1',
```